spec/queries/mentoring_query_spec.rb

```diff
@@ -36,16 +36,37 @@
       let!(:brown_mentee) { create(:user, reviewer_id: brown.id) }
       let!(:other_brown_mentee) { create(:user, reviewer_id: brown.id) }
 
-      it 'returns mentors' do
-        expect(mentors).to contain_exactly(bob.name, brown.name)
+      context 'and mentees are active' do
+        it 'returns mentors' do
+          expect(mentors).to contain_exactly(bob.name, brown.name)
+        end
+
+        it "returns mentors's offices" do
+          expect(offices).to contain_exactly(bob.office.city, brown.office.city)
+        end
+
+        it "returns mentor's mentees" do
+          expect(mentees.flatten).to contain_exactly(bob_mentee.name, other_bob_mentee.name, brown_mentee.name, other_brown_mentee.name)
+        end
       end
 
-      it "returns mentors's offices" do
-        expect(offices).to contain_exactly(bob.office.city, brown.office.city)
-      end
+      context 'and mentee are inactive' do
+        before do
+          bob_mentee.update(active: false)
```

Comment:
is better to create an inactive, with this update you are hitting de DB twice

Reply:
I'll do this changes.